package.json

```diff
@@ -19,11 +19,13 @@
   },
   "homepage": "https://github.com/adobe/aem-block-collection#readme",
   "devDependencies": {
-    "@babel/eslint-parser": "7.26.8",
-    "eslint": "8.57.1",
-    "eslint-config-airbnb-base": "15.0.0",
```

Comment:
the airbnb config hasn't been updated in 3+ years and is not supported by ESLint v9

moved to JS base which seems similar